### ISSUE TITLE
Regression: Font properties aren't applied when changed

### ISSUE THREAD

**Issue by mike-marcacci** [react-native-community/react-native-svg#760]
When font properties such as `fontSize` are changed, the new value is not applied to the image. Note that *if another non-font property is changed, such as `fill`* the new font properties are applied.

This effects version 6.3 and newer, but worked correctly in 6.2.2. Therefore, **[running this example on snack](https://snack.expo.io/SkbgqanLm) shows the expected behavior, since expo still uses 6.2.2**:

```ts
// @flow

import React, { PureComponent } from "react";
import { View, Button } from "react-native";
import Svg, { Text } from "react-native-svg";

export default class extends PureComponent<
  {},
  { fontSize: number, fill: string }
> {
  state = {
    fontSize: 20,
    fill: "red"
  };

  _toggleFontSize = () => {
    this.setState(({ fontSize }) => ({
      fontSize: fontSize === 20 ? 40 : 20
    }));
  };

  _toggleFontSizeAndFill = () => {
    this.setState(({ fontSize, fill }) => ({
      fontSize: fontSize === 20 ? 40 : 20,
      fill: fill === "red" ? "blue" : "red"
    }));
  };

  render() {
    return (
      <View>
        <Svg
          style={{ backgroundColor: "rgb(230, 230, 230)" }}
          width={400}
          height={200}
        >
          <Text
            x={0}
            y={100}
            fill={this.state.fill}
            fontSize={this.state.fontSize}
          >
            Some example text
          </Text>
        </Svg>

        <View>
          <Button onPress={this._toggleFontSize} title="Toggle Font Size" />
          <Button
            onPress={this._toggleFontSizeAndFill}
            title="Toggle Font Size & Fill"
          />
        </View>
      </View>
    );
  }
}
```

**Comment by mike-marcacci** [comment]
Here is the actual behavior, using `v6.5.2`:

![screencast 2018-08-23 17_06_36](https://user-images.githubusercontent.com/690517/44558071-fce79080-a6f6-11e8-998f-8f848044f767.gif)

**Comment by msand** [comment]
Thanks for the helpful reproduction. I think I might have found a solution. Can you try this patch? https://github.com/react-native-community/react-native-svg/pull/757/commits/d0a9cbaceb71d29d55d8e0f675d7d1608cb99ad2
@msageryd I think this might interest you as well.

**Comment by msand** [comment]
Same fix for android https://github.com/react-native-community/react-native-svg/pull/757/commits/8e0420fc2d2f5ab58f967a28c03b1dd75bdc5f6e

**Comment by mike-marcacci** [comment]
@msand – wow! Thanks for the super fast fix! I tested this patch out it works great.

Quick (unrelated) curiosity: your linked commit is clearly hosted on github, but doesn't appear to be part of any refs (branch or tag), which makes it challenging to fetch. Are you working on a hidden branch? I've just never seen this workflow before!

**Comment by msand** [comment]
It's part of this PR https://github.com/react-native-community/react-native-svg/pull/757
It's in my fork in the NativeAnimationExperiment branch. As I haven't merged the PR it's not visible in any branch of this repo yet. Good to know it works! 👍

**Comment by msageryd** [comment]
@msand I installed your patch (d0a9cba), but unfortunately this didn't solve my two problems. I checked the code in node_modules to ensure that I got your patch. I also deleted ios/build to ensure no cache problems prevented your code to build.

I hope I can find time this week to set up a new app project and try to recreate my problems as cleanly as possible.

**Comment by msand** [comment]
I've release the fixes, latest version now is v7.0.1 
This can probably closed now.
